commands/minecraft.js

```diff
@@ -0,0 +1,44 @@
+const Augur = require("augurbot"),
+  request = require("request-promise-native"),
+  minecraft = require("../utils/minecraftAPI");
+
+const Module = new Augur.Module()
+  .addCommand({
+    name: "minecraftskin",
+    description: "Gets the Minecraft skin of a user.",
+    syntax: "[@user]",
+    category: "Minecraft",
+    process: async (msg, suffix) => {
+      let user = false,
+        name = false;
+
+      if (u.userMentions(msg).size > 0) {
+        user = u.userMentions(msg).first();
+      } else if (!suffix) {
+        user = msg.author;
+      }
+
+      if (user) {
+        let ign = await Module.db.ign.find(user.id, 'twitch');
```

Comment:
Searching for minecraft names, not Twitch
```suggestion
        let ign = await Module.db.ign.find(user.id, 'minecraft');
```